### PR TITLE
:es6: Avatar API

### PR DESCRIPTION
**Pull request by regiskuckaertz**
@desbo Including you because I believe you worked on migrating AB tests?

I got validation errors for one test and had to perform a few modifications.

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js [hunk 1]
```diff
@@ -171,27 +171,24 @@ define([
...
-
-        this.componentName = 'mem_acquisition_' + trackingCampaignId + '_' + this.id;
-
-        this.template = options.template || controlTemplate;
-
-        this.blockEngagementBanner = options.blockEngagementBanner || false;
-        this.engagementBannerParams = options.engagementBannerParams || {};
-
-        this.isOutbrainCompliant = options.isOutbrainCompliant || false;
+        this.options = {
```
Comment (desbo) [inline static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js]: there are a few places outside of this file that access these properties, so will need updating to use `options`

**Review by desbo — APPROVED** [review]
👍

File: static/src/javascripts/projects/common/modules/experiments/acquisition-test-selector.js [hunk 2]
```diff
@@ -51,9 +51,10 @@ export const getTest = () => {
             count: maxViewCount,
             days: maxViewDays,
             minDaysBetweenViews: minViewDays,
-            isUnlimited,
         } = variant.options.maxViews;
 
+        const isUnlimited = variant.options.isUnlimited;
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/experiments/acquisition-test-selector.js]: ha! good catch

**Comment by regiskuckaertz** [comment]
Thanks a lot for going through the AB test @desbo 🎩

**Pull request merged**